pogoiv/iv_calculator.py

```diff
@@ -77,13 +77,45 @@ def get_ivs(self, pokemon_name, current_cp, current_health, dust_to_upgrade, pow
 
         return possible_stats
 
-    def get_ivs_across_powerups(self, pokemon_name, powerup_stats):
+    def appraisal_filter(self, response, appraisal):
+        filtered_response = []
+        """ Filters the resulting iv set according to the appraisal values. """
+        """ Global appraisal ranges. """
+        percentajes = [(0.0,48.9), (51.1,64.4), (66.7,80.0), (82.2,100.0)]
+        min_percentaje = percentajes[appraisal[0]][0]
+        max_percentaje = percentajes[appraisal[0]][1]
```

Comment:
A slightly easier to understand / cuter form of the expression worth considering:

`min_percentage, max_percentage = percentages[appraisal[0]]`